tests/blockchain/test_blockchain.py

```diff
@@ -3611,3 +3611,50 @@ async def test_reorg_flip_flop(empty_blockchain, bt):
 
     for block in chain_b[40:]:
         await _validate_and_add_block(b, block)
+
+
+@pytest.mark.parametrize("num_plot_filters_disallowed_to_pass", [1, 2])
+@pytest.mark.parametrize("bt_respects_soft_fork3", [True, False])
+@pytest.mark.parametrize("soft_fork3_height", [0, 10, 10000])
+@pytest.mark.asyncio
+async def test_soft_fork3_activation(
+    blockchain_constants, bt_respects_soft_fork3, soft_fork3_height, db_version, num_plot_filters_disallowed_to_pass
```

Comment:
it looks a bit odd to use `blockchain_constants` in addition to parameterizing on `soft_fork3_height`. The constants are already parameterized on the `Mode`, and you have `Mode.SOFT_FORK3` already